pkg/controller/controller.go

```diff
@@ -65,7 +66,13 @@ func (c *Controller) Start() error {
 		return nil
 	}
 
+	secertManager, err := security.NewsecretManager()
+	if err != nil {
+		return fmt.Errorf("secretManager create failed: %v", err)
+	}
+
 	c.client = NewXdsClient(c.mode, c.bpfWorkloadObj)
+	c.client.workloadController.Processor.Sc = secertManager
```

Comment:
In go, we donot usually use `Sc` for abbrev

Reply:
change to Sm